python/cugraph-pyg/cugraph_pyg/tests/conftest.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2022-2023, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import tempfile
+
+import pytest
+
+from dask.distributed import Client
+from dask_cuda import LocalCUDACluster
+from dask_cuda.initialize import initialize
+
+from cugraph.dask.comms import comms as Comms
+from cugraph.dask.common.mg_utils import get_visible_devices
+
+
+# module-wide fixtures
+
+
+# Spoof the gpubenchmark fixture if it's not available so that asvdb and
```

Comment:
[This ](https://github.com/rapidsai/cugraph/blob/1cb20d10b64bd00537b4fa479953d07ee90c6abb/benchmarks/cugraph/pytest-based/bench_cugraph_uniform_neighbor_sample.py#L23-L33)is another pattern we've been using (although it still requires `pytest-benchmark` which is much more common).

Reply:
changed to the new pattern